packages/block-library/src/template-part/edit/inner-blocks.js

```diff
@@ -14,7 +14,8 @@ export default function TemplatePartInnerBlocks() {
 	);
 	return (
 		<InnerBlocks
-			__experimentalBlocks={ blocks }
+			// Allow initial blocks from a block transform to load, if any.
+			__experimentalBlocks={ blocks.length === 0 ? undefined : blocks }
```

Comment:
this should be `value={ }` now :) I'm also not sure if it's still needed with the latest block sync changes